### DISCUSSION TITLE
Alias Signature

### DISCUSSION THREAD

**Discussion by trikotret** [simple-login/app#671] [Feature Requests]
Since we can have display names for aliases, can we have custom signatures for aliases too? Just like each alias can have its own display sender name, it be nice to have signatures for these aliases.

**Comment by nguyenkims** [comment]
@trikotret do you mean the signature will be appended automatically to the body of emails sent out from the alias? Modifying body of an email is actually quite error prone and might break the email format.

**Reply by trikotret** [reply]
Yes. I guess its not possible now then if it breaks the email format.

**Reply by JtwoA** [reply]
We already have the ability for a single signature - having a manual signature picker and the ability to have 3-5 signatures would solve @trikotret's issue.

**Comment by WeboR6** [comment]
Good idea! Supporting

**Comment by LavaBit5** [comment]
It will be really good if It will be implemented. It will help to set signature for aliases that will allow to split work and personal aliases
